src/graphql/main/schema.graphql

```diff
@@ -346,6 +346,8 @@ type Mutation {
   ): LnInvoicePaymentSendPayload!
   onChainAddressCreate(input: OnChainAddressCreateInput!): OnChainAddressPayload!
   onChainAddressCurrent(input: OnChainAddressCurrentInput!): OnChainAddressPayload!
+  captchaCreateChallenge: CaptchaCreteChallengePayload!
```

Comment:
Typo. Should be `CaptchaCreateChallengePayload`

Reply:
Good catch. Fixed.